src/Connector.js

```diff
@@ -59,6 +59,11 @@ class Connector {
     }, { consumerTag: `${thingId}-update` });
   }
 
+  async clearListeners(thingId) {
+    await this.client.amqp.unsubscribeConsumer(`${thingId}-request`);
```

Comment:
This isn't good because we are accessing client's internal properties directly. The ideal is to create an `unsubscribe(topic)` operation on the SDK.